### PR TITLE
fix: allow space in cafe css project step 34 

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Ref https://github.com/freeCodeCamp/freeCodeCamp/issues/52549

Okay I believe everything is in order now.

### CODE REVIEW

**Review by moT01 — APPROVED** [review]
LGTM 🎉

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f76967fad478126d6552b0d.md [hunk 1]
```diff
@@ -14,19 +14,22 @@ Next, you want to align the price to the right. Add a class named `price` to you
 You should add the `price` class to your `p` element.
 
 ```js
-assert(code.match(/<p\s*class=('|")price\1\s*>/i));
+var el = document.querySelector('.price');
+assert(!!el);
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f76967fad478126d6552b0d.md:18]:
```suggestion
assert.exists(el);
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f76967fad478126d6552b0d.md [hunk 2]
```diff
@@ -14,19 +14,22 @@ Next, you want to align the price to the right. Add a class named `price` to you
 You should add the `price` class to your `p` element.
 
 ```js
-assert(code.match(/<p\s*class=('|")price\1\s*>/i));
+var el = document.querySelector('.price');
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f76967fad478126d6552b0d.md:17]:
```suggestion
const el = document.querySelector('.price');
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f76967fad478126d6552b0d.md [hunk 3]
```diff
@@ -14,19 +14,22 @@ Next, you want to align the price to the right. Add a class named `price` to you
...
+assert(!!el);
 ```
 
 You should only have one element with the `price` class.
 
 ```js
-assert($('.price').length === 1);
+var elements = document.querySelectorAll('.price');
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f76967fad478126d6552b0d.md:24]:
```suggestion
const elements = document.querySelectorAll('.price');
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f76967fad478126d6552b0d.md [hunk 4]
```diff
@@ -14,19 +14,22 @@ Next, you want to align the price to the right. Add a class named `price` to you
...
+assert(elements.length === 1);
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f76967fad478126d6552b0d.md:25]:
```suggestion
assert.lengthOf(elements, 1);
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f76967fad478126d6552b0d.md [hunk 5]
```diff
@@ -14,19 +14,22 @@ Next, you want to align the price to the right. Add a class named `price` to you
...
 ```
 
 Your `price` class should be on the `p` element with the text `3.00`.
 
 ```js
-assert($('.price')[0].innerText.match(/3\.00/i));
+var el = document.querySelector('.price');
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f76967fad478126d6552b0d.md:31]:
```suggestion
const el = document.querySelector('.price');
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f76967fad478126d6552b0d.md [hunk 6]
```diff
@@ -14,19 +14,22 @@ Next, you want to align the price to the right. Add a class named `price` to you
...
+assert(el.innerText.match(/3\.00/i));
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f76967fad478126d6552b0d.md:32]:
```suggestion
assert.match(el.innerText, /3\.00/i);
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Two things:

- We should not use `var` to declare variables in the tests. We can use `const`, since the variables aren't reassigned (the tests run in isolation).
- Using specific `assert` API methods give us better errors when the CI fails.

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**